lib/dialects/abstract/query-generator.js

```diff
@@ -560,6 +560,91 @@ const QueryGenerator = {
     return Utils._.compact(ind).join(' ');
   },
 
+  addConstraintQuery(tableName, options) {
+    options = options || {};
+    const fieldsSql = options.fields.map(field => {
+      if (typeof field === 'string') {
+        return this.quoteIdentifier(field);
+      } else if (field._isSequelizeMethod) {
+        return this.handleSequelizeMethod(field);
+      } else {
+        let result = '';
+
+        if (field.attribute) {
+          field.name = field.attribute;
+        }
+
+        if (!field.name) {
+          throw new Error('The following index field has no name: ' + util.inspect(field));
+        }
+
+        result += this.quoteIdentifier(field.name);
+        return result;
+      }
+    });
+
+    if(options.where) {
```

Comment:
Thanks for catching. I'll fix all occurences and also add `keyword-spacing` to eslintrc